src/components/UI/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent, MouseEventHandler } from 'react';
+import classes from './Button.module.css';
+
+type ButtonProps = {
+  onClick?: MouseEventHandler
+}
+
+// ToDo: create disable mode (with appropriate style)
+const button: FunctionComponent<ButtonProps> = props => {
```

Comment:
React hint: `FunctionComponent` has also a shorthand alias `FC`

Reply:
Convention hint: React components are named in UpperCase, and I recommend exporting them directly (`export const CromwellButton`), so the user of your API doesn't need to invent names for the components.

Reply:
The constants were renamed in the appropriate style and exported directly